front/src/modules/timesStops/styles/_timesStopsDatasheet.scss

```diff
@@ -19,8 +23,20 @@
     color: var(--grey50);
   }
 
-  .invalidCell {
-    color: red !important;
+  .self-center {
+    align-self: center;
+  }
+
+  .small-unit-container {
+    display: inline-block;
+    font-size: 0.5rem;
+    line-height: 0.5rem;
```

Comment:
Can you use px for line-height for the moment ? :) Maybe it's not very common, but it's our guideline for the moment

Reply:
Sure! I did put it in px in the meantime